### PR TITLE
Add github actions to Jobs projects.

### PR DESCRIPTION
**Pull request by pwei1018**
*Issue #:* https://github.com/bcgov/entity/issues/3261

*Description of changes:*
1. add ci/cd actions to jobs projects.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

File: data-reset-tool/k8s/templates/data-reset-tool-deploy.json [hunk 1]
```diff
@@ -86,7 +86,7 @@
                                         "name": "DATABASE_USERNAME",
                                         "valueFrom": {
                                             "secretKeyRef": {
-                                                "name": "postgresql-${TAG_NAME}",
+                                                "name": "data-rest-tool-${TAG_NAME}-secret",
```
Comment (thorwolpert) [inline data-reset-tool/k8s/templates/data-reset-tool-deploy.json:89]: should this be data-reset?

**Review by kialj876 — APPROVED** [review]
Looks good!

File: .github/workflows/data-reset-tool-cd.yml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      - name: Set env by input
+        run: |
+          echo "::set-env name=TAG_NAME::${{ github.event.inputs.environement }}"
+
+      - name: Set env by master branch
+        if: env.TAG_NAME == '' && github.ref == 'refs/heads/master'
+        run: |
+          echo "::set-env name=TAG_NAME::dev"
+
+      - uses: pwei1018/bcrs-cd-action@latest
```
[the remark this answers is not in the document]
Comment (pwei1018) [reply .github/workflows/data-reset-tool-cd.yml:31]: Yes, we will. I am waiting some updates (support conditions) from github of composite event. Currently, it is still on an early stage for this action. A lots of update is on going.

File: data-reset-tool/k8s/templates/data-reset-tool-deploy.json (hunk 1 above)
Comment (pwei1018) [reply data-reset-tool/k8s/templates/data-reset-tool-deploy.json]: Fixed.

**Review by thorwolpert — APPROVED** [review]
sweet

**Pull request merged**